package.json

```diff
@@ -41,8 +44,11 @@
     "babel-plugin-tester": "5.5.2",
     "classnames": "2.2.6",
     "css-loader": "1.0.1",
+    "cypress": "3.1.5",
```

Comment:
Because this takes a long time to install, be sure to cache it for Travis CI builds. Just need one line, see:
- https://changelog.travis-ci.com/for-node-js-builds-cache-npm-shortcut-is-available-77295
- https://docs.cypress.io/guides/guides/continuous-integration.html#Travis